published/012-2016-August-24.md

```diff
@@ -0,0 +1,110 @@
+# IPFS Newsletter #11
+
+[IPFS](//ipfs.io/) is a new hypermedia distribution protocol, addressed by content and identities, aiming to make the web faster, safer, and more open. In these posts, we highlight some of the development that has happened since the IPFS team week in Lisbon in late July. This is the first Newsletter in a while, and there's a ton of exciting content below.
+
+For anyone looking to get involved, follow the embedded hyperlinks, search the wealth of information on [GitHub](//github.com/ipfs) or join us on [IRC](//webchat.freenode.net/?channels=ipfs) (#ipfs on the Freenode network).
+
+If you would like to get this update as an email, sign up for our [newsletter](//tinyletter.com/ipfsweekly)!
+
+## Updates
+
+### [Orbit](https://github.com/haadcode/orbit)
+
+![uPort](orbit-uport.gif)
+
+Orbit, the decentralized chat app run on top of IPFS, has just integrated uPort identities. There's a writeup on what uPort is [on Medium](https://medium.com/@ConsenSys/uport-the-wallet-is-the-new-browser-b133a83fe73) - essentially, it is a web-based wallet and identity management system. This means that users will be able to use their uPort name and profile picture in Orbit, as well as provision a signing key that will sign Orbit messages. We've been working with [ConsenSys](https://consensys.net/) to make this happen. You can track this [here](https://github.com/haadcode/orbit/issues/37). 
+
+![textui](orbit-textui)
+
+Orbit also now has a [textui](https://github.com/haadcode/orbit-textui). This uses [blessed](https://github.com/chjj/blessed), a curses-like library with a high level terminal interface API for Node.js. This means you can chat to people using Orbit, straight from where you get your work done, in the terminal. Of course, other venues, [like Slack](https://github.com/haadcode/orbit/issues/71), are also in the pipeline.
+
+If you're interested in getting involved with developing Orbit and helping us get there faster, we've been adding a lot more 'Help Wanted' labels to issues that we think could use some work. Look at [these issues](https://github.com/haadcode/orbit/issues?q=is%3Aopen+is%3Aissue+label%3A%22help+wanted%22+sort%3Areactions-%2B1-desc) and dive in. If you have questions, just jump on [the IRC channel](https://webchat.freenode.net/?channels=%23ipfs), or comment on the issues themselves.
+
+### [go-ipfs](//github.com/)
+
+We updated `go-ipfs` to 0.4.3-rc3 on August 8th. As we state in the [changelog](https://github.com/ipfs/go-ipfs/blob/master/CHANGELOG.md), this release candidate fixes a panic that occurs when input from stdin was expected, but none was given. Check out [this issue](https://github.com/ipfs/go-ipfs/pull/3050) for details. If you missed the 0.4.3 roll-out, you can take a look at that changelog [here]().
```

Comment:
How much we want to talk about v0.4.3 depends on the scope of the newsletter. The last release v0.4.2 is over three months old, so I'd say probably only the very recent developments of v0.4.3 are relevant (i.e. rc3).

Is the only change between rc2 and rc3 really just ipfs/go-ipfs#3050?

> If you missed the 0.4.3 roll-out, you can take a look at that changelog [here]().

Not sure what "rollout" means here -- we haven't released it yet. It's good to point to the rc1/2/3 binaries on dist.ipfs.io though :)